samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -230,6 +235,8 @@ WithButtonSpotlightGoesToCorrectTarget.parameters = {
 	}
 };
 
+select('scrollMode', WithButtonSpotlightGoesToCorrectTarget, prop.scrollModeOption, Config);
```

Comment:
Let's move this line to 231 before the definition of storyName as other sample does.